tests/integration/kubernetes/confidential_common.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bats
+# Copyright 2022-2023 Advanced Micro Devices, Inc.
+# Copyright 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+SUPPORTED_HYPERVISORS=(
+#    "qemu-sev"
+    "qemu-snp"
+    "qemu-tdx"
+)
+
+function setup_unencrypted_confidential_pod() {
+    local IMAGE_REPO="ghcr.io/confidential-containers/test-container"
+    local UNENCRYPTED_IMAGE_URL="${IMAGE_REPO}:unencrypted"
+    local TEST_DIR="$(mktemp -d /tmp/test-kata-confidential.XXXXXXXX)"
```

Comment:
I think it's better to handle temp directory creation directly in the bats. This way you can clean it up easier. The tests repo snp.bats has a good example for how to set this up:
https://github.com/kata-containers/tests/blob/4fec2b5554ea8c14992c82854776d91b8660860f/integration/kubernetes/confidential/snp.bats#L30

Reply:
Yes, good idea.

Reply:
We are now using `setup_file` which should take care of this.